scripts/detector.py

```diff
@@ -10,7 +10,13 @@
 from geometry_msgs.msg import Pose2D
 from cv_bridge import CvBridge, CvBridgeError
 
-FORK_U_OFFSET = 36
+from matplotlib import pyplot as pyplot
+import numpy
+
+# FORK_U_OFFSET = 36
+# FORK_V_OFFSET = 17
+
+FORK_U_OFFSET = 39
```

Comment:
These are manually tuned params that were set by visually inspecting the offset from generated images. Can we leave a comment that it can be tuned (how?) and remove the commented out old params?